bindings/wasm/bindings.cpp

```diff
@@ -170,6 +170,8 @@ EMSCRIPTEN_BINDINGS(whatever) {
       .function("subtract", &Difference)
       .function("intersect", &Intersection)
       .function("_GetMesh", &Manifold::GetMesh)
+      .function("_GetMeshDirect", &Manifold::GetMeshDirect)
+      .function("_FreeMeshDirect", &Manifold::FreeMeshDirect)
```

Comment:
Would you mind adding the example you're using for bench-marking? You might make a copy of three.html and modify it. I'd like to see how usage of the JS side of the API looks. Also, if you can include a summary of your bench-mark results in this PR (most of them are in the discussion at the moment), that would be very helpful!

Reply:
there are several speed-ups with this approach. it avoids the construction of an intermediate nested object representation of flat arrays and all the accessor calls that go with it. it also avoids the need to re-construct the flat arrays from the intermediate object representation. this is the bulk of the speedup. not the copy. it also reduces memory pressure by avoiding the cleanup of the intermediate structures. when you take all of that into account, it's more like a 250x speedup. I wasn't including the re-encoding of arrays in the benchmark.

I posted the benchmark code in the discussion [here](https://github.com/elalish/manifold/discussions/255#discussioncomment-3946541).

Reply:
Okay, would you mind just updating the ManifoldCAD output [function](https://github.com/elalish/manifold/blob/master/bindings/wasm/examples/worker.js#L149)? That'll make it very easy for us to bench-mark a variety of uses, and of course we'll want to use your fast version here anyway.

Reply:
will do as soon as I get this PR into its final form.